addons/weave-net/weave-net.yaml

```diff
@@ -61,12 +61,12 @@ items:
     subjects:
       - kind: ServiceAccount
         name: weave-net
-        namespace: kube-system
+        namespace: weavek8sops
   - apiVersion: rbac.authorization.k8s.io/v1
     kind: Role
     metadata:
       name: weave-net
-      namespace: kube-system
+      namespace: weavek8sops
```

Comment:
I'm pretty sure this change will break the mechanism [here](https://github.com/weaveworks/weave/blob/078b76f3f1184fd2cd98b143d8cffe6e72435d01/prog/kube-utils/peerlist.go#L17) used to detect that nodes have been removed from the cluster.

Also, I can't see how this change matches the commit message - can you explain that?

Reply:
It's something magically done by WKSctl. It replaces all occurrences of `kube-system` to `weavek8sops`. I haven't checked yet which part of the codes doing this.

Reply:
(btw, I too prefer `kube-system`)

Reply:
I still can't see how changing the namespace _to_ `weavek8sops` addresses the problem that "system-node-critical priorityClass is not permitted in weavek8sops namespace".

Also it seems you removed that priority class in a subsequent commit - why is this change now needed?

Reply:
nop, changing the namespace is not to solve the problem. It solved another problem. 

I mean `wksctl` require many components, including `weave-net` to run only in the `weavek8sops` namespace.

That's way I have to drop the `priority class`.

Reply:
Add some explanation about the changes into the extended commit message.